apps/trading/pages/portfolio/withdraw/withdraw-page-container.tsx

```diff
@@ -0,0 +1,98 @@
+import { gql } from '@apollo/client';
+import { t } from '@vegaprotocol/react-helpers';
+import { Splash } from '@vegaprotocol/ui-toolkit';
+import { useVegaWallet } from '@vegaprotocol/wallet';
+import { WithdrawManager } from '@vegaprotocol/withdraws';
+import { ASSET_FRAGMENT } from '../../../lib/query-fragments';
+import Link from 'next/link';
+import { PageQueryContainer } from '../../../components/page-query-container';
+import { isERC20Asset } from '../../../lib/assets';
+import type {
+  WithdrawPageQuery,
+  WithdrawPageQueryVariables,
+} from './__generated__/WithdrawPageQuery';
+
+const CREATE_WITHDRAW_PAGE_QUERY = gql`
```

Comment:
**praise (non-blocking)**: I know it's standard now but I'm still really happy with the decision to colocate GraphQL queries.